src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -137,6 +168,7 @@ const AddInstructors = ({ isOpen, onClose }) => {
             || instructorRequest.status === RequestStatus.COMPLETE_WITH_ERRORS) && (
               <Form>
                 <FormGroup controlId="formState">
+                  {!isEditing && (
```

Comment:
Why is necessary to add this validation for each field? Is not possible to leave all inputs inside of the same conditional?